docs/contributing/pull-requests/index.md

```diff
@@ -88,43 +88,23 @@ contributor to revise their local history before pushing to the remote repositor
 is squashing multiple half-working commits. Please be sure to follow the
 [force-pushing recommendations](#force-pushing).
 
-## Creating a Pull Request
+:::warning
 
-The Bitwarden repositories have a _Pull Request template_ which should be followed. This will ensure
-the PR review goes smoothly since it will provide context to the reviewer. <community> Once a
-community PRs has been created, they will be automatically be linked to an internal Jira ticket. The
-internal ticket is used for prioritization and tracking purposes. </community>
+**Avoid force push** once a PR has been reviewed.
 
-<bitwarden>
+Git operations that affects the existing git commits prevent GitHub from correctly identifying “new
+changes” to a PR forcing the reviewer to start over again.
 
-Please follow the following checklist when creating a PR.
-
-- The PR should always contain a Jira reference.
-  - Ensures QA have visibility in what they need to test.
-  - Jira will automatically add a reference to the PR in the ticket if you include the ticket number
-    (e.g. EC-123) or a link to the ticket anywhere in the PR title or body.
-- Add appropriate reviewer(s).
-  - In most cases, this should be your team. Each team should have a GitHub team to simplify
-    requesting reviews. The teams are named as follows:
-    - `@team-{teamAbbreviation}`: This will notify the entire team, including product, design, QA
-      and development
-    - `@team-{teamAbbreviation}-dev`: This will notify or assign for code review the software
-      engineers within the team
-    - `@team-{teamAbbreviation}-qa`: This will notify or assign for code review the QA
-      engineers/SDET(s) within the team
-  - For design changes, tag `@dept-design`
-  - Please use the
-    [SME Yellowpages](https://bitwarden.atlassian.net/wiki/spaces/DEV/pages/195919928) to check if
-    there is someone with specific knowledge of the area that can assist with reviewing complex
-    changes.
+:::
 
-</bitwarden>
-
-### Force Pushing
+## Creating a Pull Request
 
-Once a PR has been reviewed, please avoid git operations that affects the existing git commits, i.e.
-anything that requires a force push. Force pushing prevents GitHub from correctly identifying “new
-changes” to a PR forcing the reviewer to start over again.
+The Bitwarden repositories have a _Pull Request template_ which should be followed. This will ensure
```

Comment:
🎨 Could link to https://github.com/bitwarden/template/blob/main/.github/PULL_REQUEST_TEMPLATE.md.

Reply:
good call, I also want to update the template to link here.

Reply:
actually, we haven't yet standardized on that template and this needs to apply across our repos. I think that addition is more appropriate once that template is emblematic of what you see when opening a PR to one of our production repositories

Reply:
This is currently set up to be the template repository for new repositories in our organization. A collection of engineers reviewed the new PR template and approved it a couple months ago. If there's a desire for further changes it can be done over in the repository, but I do consider it the official template at this point. It's always open for improvements.

Reply:
I'm not saying it's not official, just that when you open a PR, you don't see that template today. We can (and should) update this to link when we have updated those templates

Reply:
Well, do you mean when it's everywhere? It's in a good number of repos so far. I'm fine with not linking to the template repo regardless, as you'll use a PR template in all cases.